src/index.html

```diff
@@ -6,9 +6,56 @@
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <container class="container">
```

Comment:
Consider using existing tags for containers. The HTML should be valid
<img width="1085" alt="Screenshot 2023-08-01 at 18 15 44" src="https://github.com/mate-academy/layout_moyo-header/assets/135232001/eca334a3-b0d4-469a-bb3b-1941ca9c4336">